### DISCUSSION TITLE
Forwarding / reversing by 1 frame

### DISCUSSION THREAD

**Discussion by BeanXOfficial** [f1multiviewer/issue-tracker#42] [Feature requests] [answered]
If you're a bit of a nerd like me, you sometimes like to really analyse something that happens during a race or just look at a car's design features. It would be really useful if you could go frame by frame, for example by using (.) & (,) just like you would on YouTube.

**Comment by f1multiviewer** [comment, marked as answer]
This is added in the next release (beta.11)
